website/docs/language/backend/azurerm.mdx

```diff
@@ -580,10 +580,14 @@ When authenticating using a Service Principal with a Client Certificate - the fo
 
 * `client_id` - (Optional) The Client ID of the Service Principal. This can also be sourced from the `ARM_CLIENT_ID` environment variable.
 
+* `client_id_file_path` (Optional) The path to a file containing the Client ID which should be used. This can also be sourced from the `ARM_CLIENT_ID_FILE_PATH` Environment Variable.
```

Comment:
```suggestion
- `client_id_file_path`: (Optional) Specifies the path to a file containing the client ID. Terraform presents the client ID when authenticating with Azure. You can set the `ARM_CLIENT_ID_FILE_PATH` environment variable to configure this option.
```
What does Terraform do with the client ID? Please correct this suggestion as necessary.

Reply:
> Terraform presents the client ID when authenticating with Azure